src/app/treasure-hunt/treasure-hunt-report/treasure-hunt-ppt/treasure-hunt-ppt.service.ts

```diff
@@ -347,6 +347,40 @@ export class TreasureHuntPptService {
         }
         slideTeamTopOpps.addTable(rows, { x: 0.14, y: 2.5, w: 11.05, colW: [2, 1.5, 1.5, 0.8, 1.25, 1.25, 1.25, 1.25, 1.25, 1], color: "1D428A", fontSize: 12, fontFace: 'Arial (Body)', border: { type: "solid", color: '1D428A' }, fill: { color: 'BDEEFF' }, align: 'left', valign: 'middle' });
 
+        let slideTeam3TopOppsCostSaving = pptx.addSlide({ masterName: "Title Only" });
```

Comment:
A few things can be simplified in this block:
- top3Rows array initialization with the defaults
- Array.filter can be used
- Array.slice could be used